templates/web/base/my/planned.html

```diff
@@ -29,6 +29,12 @@ <h1>[% loc('Your shortlist') %]</h1>
 <div id="js-reports-list">
 [% INCLUDE 'my/_problem-list.html' shortlist = 1 %]
 </div>
+<div class="shadow-wrap">
```

Comment:
Could we have this only appear if JS is working? Sticking a "hidden-nojs" on this div should do it.